.github/workflows/bundle-size-analysis.yml

```diff
@@ -0,0 +1,107 @@
+name: "Bundle size"
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # Build current PR and upload it's bundle stats file
+  build-pr:
+    name: "Building Pull Request"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+    
+    steps:
+    - uses: actions/checkout@v3.0.0
+      with:
+        ref: ${{github.event.pull_request.head.ref}}
+    
+    - uses: actions/setup-node@v3.4.1
+      with:
+        node-version: 16
+
+    - name: Install dependencies
+      uses: bahmutov/npm-install@v1.6.0
+      with:
+       install-command: npm install --force

```

Comment:
`--force` is a bad practice but it fails without it, with errors of pdf-dist, pdf worker not resolving.